subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -106,6 +106,16 @@
             "changes": [
                 "Interface has been removed"
             ]
+        },
+        {
+            "type": "org.gradle.tooling.UnsupportedVersionException",
```

Comment:
❌ We shouldn't add internal methods to the public API.

Reply:
Thanks. Reverted.